src/views/CollectionIndex/scss/index.scss

```diff
@@ -113,7 +113,7 @@
 
 .inner-page-wrapper {
   padding: 17px;
```

Comment:
For units like margin, padding, you should use the $spacer variable instead or use rem units to keep things consistent.